<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gradient</class>
 <widget class="QWidget" name="gradient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>图像梯度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:8px solid rgb(0, 170, 0);
background-color: rgb(198, 198, 198);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Sobel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scharr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Laplacian</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>核大小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
	border:3px solid rgb(0,0,0);
	border-radius:8px
}
#pushButton:hover{
	background-color:rgb(255,255,255);
	color:rgb(0,0,0);
}
#pushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>spinBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
